app/classifier.py

```diff
@@ -1,153 +1,337 @@
 import json
 import logging
 import os
-import string
-import time
-from typing import List, Tuple, Dict, Optional
+from datetime import datetime
+from typing import Any, Dict, List, Set
 
 import numpy as np
 import tensorflow as tf
 from flask import Blueprint
 from flask import current_app as app
 from flask import jsonify, request
+from ming import schema
+from ming.odm import FieldProperty, MappedClass, Mapper
 
-from app.embedder import Embedder, MAX_SEQ_LENGTH
+from app.db import hasher, session
+from app.embedder import MAX_SEQ_LENGTH, Embedder
 from app.model_config import InstanceConfig
-from app import model_fetcher
 
 classify_bp = Blueprint('classify_bp', __name__)
 
 
+class ClassificationSample(MappedClass):
```

Comment:
Now that we have two, can you move this and the Embedding mongo model into a /models directory so that they end up in one place?